File: Backend/ServerBackend/src/main/java/com/fran/dev/potjera/serverbackend/models/gamesession/coinbooster/CoinBoosterPlayerState.java
package com.fran.dev.potjera.serverbackend.models.gamesession.coinbooster;

import lombok.Builder;
import lombok.Data;

import java.util.List;

@Data
@Builder
public class CoinBoosterPlayerState {
    Long playerId;
    Integer correctAnswers;
    Boolean isFinished = false;
}

File: AndroidApp/app/src/main/java/com/fran/dev/potjera/android/app/game/models/state/CoinBoosterPlayerState.kt
package com.fran.dev.potjera.android.app.game.models.state

import com.fran.dev.potjera.android.app.game.models.CoinBoosterQuestion

data class CoinBoosterPlayerState(
    val playerId: Long = 0,
    val correctAnswers: Int = 0,
    val questions:List<CoinBoosterQuestion> = emptyList(),
    val isFinished: Boolean = false
)
